ui/app/pages/create-account/connect-hardware/index.scss

```diff
@@ -132,6 +113,14 @@
     color: #9b9b9b;
     margin-top: 10px;
     margin-bottom: 15px;
+
+    &__link {
+      @include H6;
+
+      text-align: center;
+      color: #2f9ae0;
```

Comment:
Ideally we'd stick to using colours that are defined in our design system, in `ui/app/css/design-system/colors.scss`.

This exact colour doesn't seem to be there though :thinking:. Was this taken from the designs? Perhaps we can use one of the other types of blue instead.

Reply:
Good shout!